### PR TITLE
Fix #4363: rename mpi param

### LINKED ISSUE #4363: template_common.get_exec_parameters_cmd param mpi conflicts
`def get_exec_parameters_cmd(mpi=False):`

need to rename param from mpi to is_mpi or whatever.

### PR DESCRIPTION
**Pull request by gurhar1133**
Renamed the param and changed calls
Anything more @robnagler?

### CODE REVIEW

File: sirepo/template/template_common.py [hunk 1]
```diff
@@ -391,10 +391,10 @@ def generate_parameters_file(data, is_run_mpi=False):
     return render_jinja(None, v, name='common-header.py'), v
 
 
-def get_exec_parameters_cmd(mpi=False):
+def get_exec_parameters_cmd(is_mpi=False):
     from sirepo import mpi as mpi_m
```
Comment (robnagler) [inline sirepo/template/template_common.py:395]: The "as mpi_m" can now be removed and mpi can be used on 397.

**Review by robnagler — CHANGES_REQUESTED** [review]

**Comment by gurhar1133** [comment]
Accidentally closed then reopened this PR

**Comment by robnagler** [comment]
does this need a review? please request one if so

**Comment by gurhar1133** [comment]
> does this need a review? please request one if so

I think so since changes requested are blocking merge. Just re-requested review.

**Review by robnagler — APPROVED** [review]

**Pull request merged**